### PR TITLE
Import accessions data

### PR DESCRIPTION
**Pull request by constanzauanini**
This has a lot of repeated code. I could try to make a component to reuse code.

I don't see accessions created after import, and I couldn't test the repeated accession number either.

Screen Shot 2022-09-26 at 11 52 11
Screen Shot 2022-09-26 at 11 51 49
Screen Shot 2022-09-26 at 11 52 03

### CODE REVIEW

File: src/components/seeds/database/index.tsx [hunk 1]
```diff
@@ -407,33 +410,111 @@ export default function Database(props: DatabaseProps): JSX.Element {
...
+          <SelectSeedBankModal
+            organization={organization}
+            open={selectSeedBankModalOpen}
+            onClose={onSeedBankSelected}
+          />
+          <SelectSeedBankModal
+            organization={organization}
+            open={selectSeedBankForImportModalOpen}
+            onClose={onSeedBankForImportSelected}
+          />
```
Comment (karthikbtf) [inline src/components/seeds/database/index.tsx:516]: lets try to use one modal? we should be able to use the correct callbacks based on which flow triggered the modal?

File: src/components/seeds/database/index.tsx [hunk 2]
```diff
@@ -407,33 +410,111 @@ export default function Database(props: DatabaseProps): JSX.Element {
...
+        id={id}
+        open={open}
+        anchorEl={anchorEl}
+        onClose={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left',
+        }}
+      >
+        <Box>
```
Comment (karthikbtf) [inline src/components/seeds/database/index.tsx:467]: looks a bit strange? 
Terraware App 2022-09-26 08-02-29

File: src/components/seeds/database/ImportAccessionsModal.tsx [hunk 3]
```diff
@@ -0,0 +1,354 @@
+import { Link } from '@mui/material';
```
Comment (karthikbtf) [inline src/components/seeds/database/ImportAccessionsModal.tsx:1]: yes if you want to build out a reusable component that would be cool, if you want to do that as a fast follow-up so we can start testing, that's fine too

**Review by karthikbtf — COMMENTED** [review]
not sure why you aren't seeing accessions,,, where did you get the accessions csv from? can you share with me?

**Comment by constanzauanini** [comment]
> not sure why you aren't seeing accessions,,, where did you get the accessions csv from? can you share with me?

I downloaded the template and added data there

File: src/components/seeds/database/ImportAccessionsModal.tsx (hunk 3 above)
Comment (constanzauanini) [reply src/components/seeds/database/ImportAccessionsModal.tsx:1]: Yes, lets merge this first and I will create a component for the two cases in next pr

File: src/components/seeds/database/index.tsx (hunk 2 above)
Comment (constanzauanini) [reply src/components/seeds/database/index.tsx:467]: Yes, I just moved the two buttons inside the menu, but how should it look? Like a simple menu?

Comment (karthikbtf) [reply src/components/seeds/database/index.tsx:467]: SBD_V2 – Figma 2022-09-26 08-25-13

Comment (constanzauanini) [reply src/components/seeds/database/index.tsx:467]: Thanks!

**Review by karthikbtf — APPROVED** [review]

**Pull request merged**